recipes-connectivity/bluez5/bluez5_%.bbappend

```diff
@@ -52,4 +52,4 @@ FILES_${PN}_append_raspberrypi0-wifi = " ${BCM_BT_FIRMWARE}"
 
 SYSTEMD_SERVICE_${PN}_append_raspberrypi0-wifi = " ${BCM_BT_SERVICE}"
 
-RDEPENDS_${PN}_append_raspebrrypi0-wifi = " udev-rules-rpi"
+RDEPENDS_${PN}_append_raspberrypi0-wifi = " udev-rules-rpi"
```

Comment:
I know this is not related to the given change proposed. But I question pi0 specific RDEPENDS, I would think it
should be coming via MACHINE_EXTRA_RRECCOMMENDS or some such.

Reply:
I'm not in love with that mostly because it replicates a series of target-specific capabilities.  However, without introducing some new OVERRIDES tag that identifies the subset of raspberry pi devices that have built-in bluetooth devices I didn't see a nice way to clean it up, other than to move the list of dependencies out to a variable which is where I left it.

Reply:
@kraj you have a point there but the issue is the fact that bluez brings the systemd service which depends on the udev rules. So this makes bluez package dependent on the udev rules. That's why we have that RDEPENDS, at runtime, the systemd service needs the udev rules. Probably the clean way forward would be to get the service out in another recipe and make that depend on udev rules but I wonder if that would not confuse people more.

Reply:
Actually with the rework it's pi-bluetooth that depends on the device name and hence on the udev script that provides the alias.  So that should be updated.

Reply:
Or maybe not: pi-bluetooth won't work if there isn't a /dev/serial1 but it'll run without it, just erroring out.  Maybe RRECOMMENDS is a better approach, though I don't want to tackle it myself/now.

Reply:
Agreed.